### ISSUE TITLE
Figure out how to import data into LenaSYS

### ISSUE THREAD

**Issue by a22willi** [HGustavs/LenaSYS#14736]
In issue [#14688](https://github.com/HGustavs/LenaSYS/issues/14688) sanitized data from the currently live version of LenaSYS was fixed so that the insert statement runs properly. Figure out how this data can be imported into the current development version of LenaSYS. Ask me or @b22marah for the data and we will provide it for you on a USB-stick. 

In the case that you cant import the data, document the issues you encounter and create new issues for fixing these. 

**IMPORTANT:** Read the previously linked issue, it contains a lot of valuable information about the state of this data.

**Comment by a22natth** [comment]
# Documentation on issue #14736
## What seems to be working:
diagram.php seems to be working as intended BUT the file cannot be saved through the save icon (although we don't know if this is beyond the scope of group 3). Additionally, preview.php seems to be working and presents no errors(as seen below).
![image](https://github.com/HGustavs/LenaSYS/assets/129295853/df444901-b586-4019-a1c3-acecf8f66934)

## What seems to NOT be working:
By trying to access the different php-pages, the 500 error is presented. 
courseed.php does not work at all, 500 (Internal Server Error appeared, see example below).

![eMSjlFnH8jSncxMslfrqPZMrg5xPf0nYDnPSubIFAIjTVgcyF0DvnXxZLzKjB_6qO8hqvuKNYGAS4YqAYH57PQV07k5_OcEA4vplfbZG2P8-SElCvwjidZ0RdWV_](https://github.com/HGustavs/LenaSYS/assets/129295853/55701d97-e6e0-46af-a082-3c692de7f540)

resulted.php seems to display a similar error. Normally, it looks like this:
![0-u173AmFI1SCbOCTRW9RUj5Fs8u-ccjsK8_rsg6nvRHabRGDIZ1GA8OSuJLqdQZ4Baw7dYYIm4D3mOkwD43MOx6uGB0ASx-RUOmFqoFvlg9j-LAF50E01MZKf6e](https://github.com/HGustavs/LenaSYS/assets/129295853/e2dda60a-5f75-4bb6-ab5a-de6e5304a211)

After the old database is ran:
![7NSrrQ6a1VyksPZtMD7y9Occ4R9JLv9EK7cZSmgIX120OolP-QfHr93EHrKZ7DsTzEDiV_l4Crp_Q2oef3Y1X8G44iEiJBQE0_UF4slsEhXyQZG5Qk4PszoghHs5](https://github.com/HGustavs/LenaSYS/assets/129295853/a81921db-3ba0-42d5-ab90-d1d56bd6150a)

sectioned.php presents these errors, which is probably due to “courseGitURL” not being a field in the old database. Normally, it looks like this;
![_qCb40WP1nb8UEjqjSawVyuNha6wI-uXEWCkGv52r7JZ21CK65T0IEHrjKLYK9r2UoX3QAJJvE1N2RG4OciH3OgHl4SEcwWNYVZcTOPB8OGXPojy7TEu7rIVo225](https://github.com/HGustavs/LenaSYS/assets/129295853/e772af26-3790-4a30-aac9-513ebd6906d3)

After the old database is ran:
![8uTQjXlXXqBsnqCVrgKgtA2oiY52oLaMBXbm62ejqIuWH66k-Udh2c7Z3ytfDnrpQ49tTZkSAkVcv6IgFpnk_5AsVPZujsU7XD3mDqhCt8yocSZ9cbVRmUM-lH18](https://github.com/HGustavs/LenaSYS/assets/129295853/86e31255-c934-4a28-9ac5-480f93773932)

testDugga.php and showDugga.php also show similar errors to the one above. We tried accessing this URL: http://localhost/LenaSYS/DuggaSys/sectioned.php?courseid=3&coursename=Datorns%20grunder&coursevers=1337 , but just as the previous attempts, the 500 error was presented yet again. We also tried to reconstruct what a link should look like with the current database data (id + coursename + course version), this had the same result.
In conclusion, the majority of the pages of LenaSYS cannot be used at all and is most likely due to the old database missing certain fields, fields being in the wrong order, or overall not being identical to the new database of LenaSYS.
